java/java-tests/testData/codeInsight/daemonCodeAnalyzer/quickFix/optionalIsPresent/afterAssignmentField.java

```diff
@@ -1,4 +1,4 @@
-// "Replace Optional.isPresent() condition with functional style expression" "true-preview"
+// "Replace Optional presence condition with functional style expression ''Optional.ifPresent()''" "true"
```

Comment:
The fix name is clearly incorrect now: it says that "Optional.ifPresent" will be used while in fact, `orElse` is used.

Reply:
Removed